### PR TITLE
OBPIH-5717 Add Haitian to the default supported locale

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: grails-app/i18n/messages.properties
Comment (jmiranda) [inline grails-app/i18n/messages.properties:3122]: Haitian isn't a language and I want these names to be in the form that would be recognized by the native speaker, so this one should be Kreyòl.

**Review by jmiranda — CHANGES_REQUESTED** [review]

File: grails-app/i18n/messages.properties
Comment (awalkowiak) [reply grails-app/i18n/messages.properties:3122]: @jmiranda I aligned it with what is returned by `Locale.getDisplayName()` here https://github.com/openboxes/openboxes/blob/develop/grails-app/views/common/_footer.gsp#LL28C9-L29C11. Otherwise we would get different labels on gsp and react footers. But if you want to turn it into `Kreyòl`, then we can change the `getDisplayName` in the `common/_footer.gsp` into `g:message`.

**Comment by awalkowiak** [comment]
@jmiranda I made some improvements on that one. You can review it.

**Comment by jmiranda** [comment]
Sorry I didn't get a chance to review this earlier today but I'm looking at it now on my phone. 

There's something with displaying the locale name that I'm not grasping and I think I need to be sitting in front of my laptop to fully understand. So give me until Tuesday before merging. 

The case I'm worried about is when a user configures more supported locales in openboxes-config.groovy.

**Comment by jmiranda** [comment]
Never mind I see that you're handling that.

**Review by jmiranda — APPROVED** [review]

File: grails-app/i18n/messages.properties
Comment (jmiranda) [inline grails-app/i18n/messages.properties:1429]: One minor change. I've been misspelling this for months. Could you change it to Acholi.

**Review by jmiranda — CHANGES_REQUESTED** [review]

**Comment by jmiranda** [comment]
Once that change is done feel free to dismiss my review and merge the PR.

**Pull request merged**
